### PR TITLE
Changed routing to activity-detail routen and fixed refresh bug on de…

### PR DESCRIPTION
**Pull request by LorenzoJokhan**
…tail page

### CODE REVIEW

File: packages/openstad-event-browser-widgets/src/components/events/event-map.tsx [hunk 1]
```diff
@@ -18,7 +23,7 @@ export function EventMap({ events }: any) {
             ]}
           >
             <Popup>
-              <Link href={`/${event.id}`}>{event.name}</Link>
+              <Link to={`${prefixUrl}${slug}/${event.id}`}>{event.name}</Link>
```
Comment (Badmuts) [inline packages/openstad-event-browser-widgets/src/components/events/event-map.tsx:26]: I don't think `to` is a prop that is available in Wouter's `<Link>` component: https://github.com/molefrog/wouter/tree/v2.7.4#link-hrefpath-

**Review by Badmuts — COMMENTED** [review]
Could you check if replacing `to` props with `href` prop still works?

**Review by Badmuts — APPROVED** [review]
🎉

File: packages/openstad-event-browser-widgets/src/components/events/event-map.tsx (hunk 1 above)
Comment (LorenzoJokhan) [reply packages/openstad-event-browser-widgets/src/components/events/event-map.tsx]: > Could you check if replacing `to` props with `href` prop still works?

Changed 'to' to href, seems to work

**Pull request merged**